src/sidebar/components/test/hypothesis-app-test.js

```diff
@@ -325,75 +363,113 @@ describe('hypothesisApp', function () {
   });
 
   describe('#logout()', function () {
-    it('calls session.logout()', function () {
-      var ctrl = createController();
-      ctrl.logout();
-      assert.called(fakeSession.logout);
-    });
 
-    it('prompts the user if there are drafts', function () {
-      fakeDrafts.count.returns(1);
-      var ctrl = createController();
+    // Tests shared by both of the contexts below.
+    function doSharedTests() {
```

Comment:
defining the tests is running them... and addSharedTests is not less ambiguous given your argument of what it is doing. But more than that, this is pretty nit picky and not something that would realistically cause anyone a road block. Let's focus on the rest